openprescribing/frontend/admin.py

```diff
@@ -79,11 +79,34 @@ class Meta:
         model = OrgBookmark
 
 
+class OrgBookmarkTypeFilter(admin.SimpleListFilter):
+    title = 'organisation type'
+    parameter_name = 'org_bookmark_type'
+
```

Comment:
I had to go check Django documentation to know how `lookups` and `queryset` are related. Maybe a short comment to explain these are used together by the base class?